package.json

```diff
@@ -2,24 +2,50 @@
   "name": "kubercade",
   "version": "1.0.0",
   "description": "Serverless containerized social arcade built on Knative.",
-  "main": "index.js",
+  "main": "build/app.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "dev": "ts-node-dev --respawn --transpileOnly ./src/app.ts",
+    "prod": "tsc && node ./build/app.js",
+    "check": "gts check",
+    "clean": "gts clean",
+    "compile": "tsc -p .",
+    "fix": "gts fix",
+    "prepare": "npm run compile",
+    "pretest": "npm run compile",
+    "posttest": "npm run check",
+    "test": "mocha --exit -r ts-node/register tests/**/*.test.ts",
+    "testWithCoverage": "nyc -r lcov -e .ts -x \"*.test.ts\" mocha -r ts-node/register tests/**/*.test.ts && nyc report"
```

Comment:
Maybe just call this `coverage`?

Reply:
Done https://github.com/knative-portability/Kubercade/pull/1/commits/44ebadc6b2b505e83f4f3a5ee84d1551a769f4fd